planning/behavior_velocity_planner/src/node.cpp

```diff
@@ -263,7 +263,9 @@ void BehaviorVelocityPlannerNode::onNoGroundPointCloud(
 
   Eigen::Affine3f affine = tf2::transformToEigen(transform.transform).cast<float>();
   pcl::PointCloud<pcl::PointXYZ>::Ptr pc_transformed(new pcl::PointCloud<pcl::PointXYZ>);
-  pcl::transformPointCloud(pc, *pc_transformed, affine);
+  if (pc.width > 0) {
```

Comment:
It  may be better to pc.width * pc.height or pc.data.empty()

Reply:
@yukkysaito I modified to use empty() at https://github.com/autowarefoundation/autoware.universe/pull/837/commits/21d72206be158cbcaa970ff812fa64a17e4e97df.